### DISCUSSION TITLE
Is it possible that detect the locale of clients on the fly?

### DISCUSSION THREAD

**Discussion by razonyang** [decaporg/decap-cms#6816] [Q&A] [answered]
I found there is an option for configuring the locale - https://decapcms.org/docs/configuration-options/#locale, it may be useful that could detect the locale automatically, such as the `navigator.language`.

**Comment by martinjagodic** [comment, marked as answer]
Hi @razonyang, this is not currently possible. If you would like to see this implemented, it would be best to submit a pull request.

**Reply by razonyang** [reply]
Thanks for the explanation, will create a PR if I find time.
